org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/quickfix/DiagnosticResolution.java

```diff
@@ -47,6 +47,16 @@ public class DiagnosticResolution {
 
 	private Options options;
 
+	@Deprecated//(forRemoval=true,since="2.27")
```

Comment:
Hmm, why is the `forRemoval` in comments?

Reply:
Cause Java 8